pom.xml

```diff
@@ -282,6 +283,49 @@
                         <link>http://fasterxml.github.io/jackson-databind/javadoc/2.6/</link>
                         <link>https://javaee-spec.java.net/nonav/javadocs/</link>
                     </links>
+                    <!-- optional dependencies from other modules (must be explicitly declared here in order to be correctly resolved) -->
+                    <additionalDependencies>
```

Comment:
Yes, it's only to centralize this rather complex configuration in one place. Granted some of these additional dependencies aren't really required in all child modules, but I think it is easier to maintain if everything is declared in one single place.